app/src/main/java/com/example/mycalculator/symbols/OperatorsSymbols.java

```diff
@@ -0,0 +1,8 @@
+package com.example.mycalculator.symbols;
+
+public class OperatorsSymbols {
+    public static final String operatorSum = "+";
+    public static final String operatorMinus = "-";
+    public static final String operatorDivide = "/";
+    public static final String operatorMultiply = "X";
```

Comment:
This ones should be on capital letter, and words separated by underscore

Reply:
Done